### PR TITLE
Small improvements to first time use of the app

### PR DESCRIPTION
**Pull request by ericcornelissen**
- Relevant Issues: #
- Related Pull Requests: -

* * *

## What
Changes in this PR:
- Renamed "FirstTimeUse" to "Welcome".
- Removed toolbar from `WelcomeActivity` and `CreateProfileActivity`.
- Added logo to `CreateProfileActivity` to make it look more like `WelcomeActivity`.
- Improved `string.xml` naming for these two activities.
- Improved JavaDoc and logging text these two activities.
- Made some small changes to the Java code of these two activities.

## Why
Mainly to improve the design of the `WelcomeActivity` and `CreateProfileActivity` activities. But while I was working on it I couldn't help but update some of the related code 😄 

## How
Uninstall the app and install it again, just check out the activities and tell me what you think 🙂 

## Alternative implementation
None

## Notes
Opening the app and seeing the design is most important when reviewing this PR. Not the actual code changes, these are generally very simple refactors (like renaming things).

### CODE REVIEW

**Comment by clenrock** [comment]
image
Could you make this logo a bit larger? Not sure if this is changed in this PR or the lint PR where we added all the new dimensions

File: app/src/main/res/layout/activity_create_profile.xml [hunk 1]
```diff
@@ -9,21 +10,14 @@
         android:layout_height="wrap_content"
         android:orientation="vertical">
 
-        <android.support.v7.widget.Toolbar
-            android:id="@+id/toolbar3"
+        <ImageView
+            android:id="@+id/large_logo"
```
Comment (jverbraeken) [inline app/src/main/res/layout/activity_create_profile.xml]: If the id isn't used you don't need to specify it, it only clutters R.java, but it's not really a problem either

File: app/src/main/res/layout/activity_create_profile.xml [hunk 2]
```diff
@@ -9,21 +10,14 @@
...
-                android:layout_height="wrap_content"
-                android:layout_gravity="center"
-                android:text="@string/app_name"
-                android:textSize="@dimen/toolbar_title_size"
-                android:textStyle="bold" />
-        </android.support.v7.widget.Toolbar>
+            android:layout_height="match_parent"
+            android:layout_gravity="center"
+            android:contentDescription="@string/logo"
+            app:srcCompat="@drawable/large_logo_with_shadow"
```
Comment (jverbraeken) [inline app/src/main/res/layout/activity_create_profile.xml]: app:srcCompat is used for vector drawables like SVG's. For all other images you should use android:src

**Comment by ericcornelissen** [comment]
@clenrock I see what you mean but I'm unable to solve it and if you look at the diff of this PR you can see that it was not introduced in this PR 😅 Feel free to play around on this branch and push a solution if you find one... Otherwise maybe open an issue for the problem.

**Pull request merged**